logic.js

```diff
@@ -36,40 +36,33 @@ http://ricostacruz.com/cheatsheets/umdjs.html
     return a;
   }
 
-  var jsonLogic = {};
+  /**
+   * Generate binary operations
+   * @param {object} operations Accumulator object for operations
+   * @param {string} operator   Binary operator to apply
+   * @return {Function}         Operator function as specified by param
+   */
+  function generateBinaryOperation(operations, operator) {
+    operations[operator] = new Function("a", "b", "return a" + operator + "b;");
```

Comment:
Isn't this equivalent of `eval` the code. I do understand the difference between `eval` & `new Function` (closure in new Function), but the concern that many users like me would have for avoiding eval would be around *Content Security Policy*

Reply:
Thanks for your feedback @computnik
Well, [`eval`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) has access to current scope and local variables, in contrast to the [`new Function` constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function) - which has not!
This difference is very often miss-understood and has plenty of discussions on the web already.
IMHO I suggest to study the ECMA specs itself carefully.
https://stackoverflow.com/questions/4599857/are-eval-and-new-function-the-same-thing

Reply:
Anyway meanwhile we forked an completely rewrote this project and I added JSON-Schemas for all operators.
https://github.com/axa-ch/json-logic-js